src/components/Layout/Section/FundingDetails.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Stats } from '~/types/Stats';
+
+interface Props {
+  title: string;
+  imageUrl?: string;
+  imageAlt?: string;
+  children: any;
+  stats?: Stats[];
+}
+
+export const FundingDetails: React.FC<Props> = ({
+  title,
+  imageUrl,
+  imageAlt,
```

Comment:
```suggestion
  imageAlt = "",
```

Jedes optionale type sollte einen default haben, oder?